apps/wakunode2/app.nim

```diff
@@ -366,9 +360,7 @@ proc startRestServer(
       "/relay endpoints are not available. Please check your configuration: --relay"
 
   ## Filter REST API
-  if conf.filternode  != "" and
-     app.node.wakuFilterClient != nil:
-
+  if conf.filternode != "" and app.node.wakuFilterClient != nil:
```

Comment:
Unnecessary nph formatting revert?

Reply:
Different version of nph maybe? `v0.5.1-0-gde5cd48` won't let me use 2 lines.